index.es6.js

```diff
@@ -1,248 +1,124 @@
 import inquirer from 'inquirer'
 import semafor from 'semafor'
-import glue from 'glue-path'
 import packager from 'electron-packager'
-import colors from 'colors'
 
-export default function interactive(verbose = true) {
-  
-  let log = semafor();
+export default function interactive() {
+  // Logging library.
+  const log = semafor()
 
-  let default_version = "0.30.4";
-  let default_src = process.cwd();
-  let default_out = glue([process.cwd(), "releases"]);
-  
-  let get_package_name = () => {
+  // Get the name from the package.json file.
+  function get_package_name() {
     try {
-      var pkg_path = glue([process.cwd(), "package.json"]);
-      var pkg = require(pkg_path);
-      return pkg.name;
-    }catch(e){}
-    return "";
+      const pkg = require(`${process.cwd()}/package.json`)
```

Comment:
Oh, didn't know Windows was OK with forward slashes, thanks for the info @davemackintosh. I'll wait the updated PR.